### ISSUE TITLE
Error: package or namespace load failed for 'covr':

### ISSUE THREAD

**Issue by eri820503** [r-lib/covr#405]
I am running Centos7 from official docker image and Microsoft R open(3.5.3).
I installed the `covr` package using `install.packages("covr")` I get the following error:
```
Error: package or namespace load failed for 'covr':
 .onLoad failed in loadNamespace() for 'covr', details:
  call: system(paste(which, shQuote(names[i])), intern = TRUE, ignore.stderr = TRUE)
  error: error in running command
Error: loading failed
Execution halted
ERROR: loading failed
* removing '/opt/microsoft/ropen/3.5.3/lib64/R/library/covr'
```
But when I am running the official R, I can install `covr` successfully.
I don't know whether `conv` doesn't support Microsoft R Open.

The completed log as following:
```
install.packages("covr")
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  136k  100  136k    0     0  95577      0  0:00:01  0:00:01 --:--:-- 95594
* installing *source* package 'covr' ...
** package 'covr' successfully unpacked and MD5 sums checked
** libs
gcc -std=gnu99 -I/opt/microsoft/ropen/3.5.3/lib64/R/include -DNDEBUG   -DU_STATIC_IMPLEMENTATION   -fpic  -DU_STATIC_IMPLEMENTATION -O2 -g  -c reassign.c -o reassign.o
gcc -std=gnu99 -shared -L/opt/microsoft/ropen/3.5.3/lib64/R/lib -o covr.so reassign.o -L/opt/microsoft/ropen/3.5.3/lib64/R/lib -lR
installing to /opt/microsoft/ropen/3.5.3/lib64/R/library/covr/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded
Error: package or namespace load failed for 'covr':
 .onLoad failed in loadNamespace() for 'covr', details:
  call: system(paste(which, shQuote(names[i])), intern = TRUE, ignore.stderr = TRUE)
  error: error in running command
Error: loading failed
Execution halted
ERROR: loading failed
* removing '/opt/microsoft/ropen/3.5.3/lib64/R/library/covr'

The downloaded source packages are in
        '/tmp/RtmpjQYjUp/downloaded_packages'
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning message:
In install.packages("covr") :
  installation of package 'covr' had non-zero exit status
```

**Comment by rohanhodarkar** [comment]
@eri820503 facing same issue. Let me know if you are able to fix it and the steps if possible.
Thanks in advance.

**Comment by jimhester** [comment]
I don't know, there isn't anything in `.onLoad()` that calls `system()` I am aware of. I don't suppose `traceback()` gives you any information...

**Comment by psehong** [comment]
I resolved this by upgrading to R 3.6.1
